scenarios/kubernetes_e2e.py

```diff
@@ -396,8 +396,7 @@ def main(args):
     if args.timeout:
         runner_args.append('--timeout=%s' % args.timeout)
 
-    cluster = args.cluster or 'e2e-gce-%s-%s' % (
-        os.environ['NODE_NAME'], os.getenv('EXECUTOR_NUMBER', 0))
+    cluster = args.cluster or 'e2e-gce-%s' % os.getenv('BUILD_NUMBER', 0)
```

Comment:
boskos hands out a new cluster for each job, right? then we shouldn't need to change the cluster name at all, and just use the default "bootstrap-e2e" that CI jobs use.